Source: Daniel Mierswa <impulze@impulze.org>
Upstream: http://lists.gnu.org/archive/html/gnutls-devel/2012-04/msg00048.html
Reason: automagic dependency on crywrap

---
 configure.ac |    9 ++++++++-
 1 file changed, 8 insertions(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index 5350a90..4ce677f 100644
--- a/configure.ac
+++ b/configure.ac
@@ -409,7 +409,14 @@ AC_CHECK_FUNCS([alarm atexit dup2 epoll_create kqueue memchr memset munmap \
 		putenv regcomp scandir select socket strcasecmp strchr \
 		strdup strerror strncasecmp strrchr strstr strtoul uname])
 
-PKG_CHECK_MODULES(LIBIDN, libidn >= 0.0.0, [libidn=yes], [libidn=no])
+AC_ARG_ENABLE(crywrap,
+	AS_HELP_STRING([--disable-crywrap], [unconditionally disable the crywrap TLS proxy service]))
+
+libidn=no
+
+if test "x$enable_crywrap" != "xno" ; then
+	PKG_CHECK_MODULES(LIBIDN, libidn >= 0.0.0, [libidn=yes], [libidn=no])
+fi
 
  if test "x$libidn" != "xno" && test "$ac_cv_func_daemon" != "no";then
   crywrap=yes
-- 
1.7.9.6

